onyo/commands/tests/test_new.py

```diff
@@ -86,6 +86,64 @@ def test_new_top_level(repo: OnyoRepo) -> None:
     fsck(repo)
 
 
+@pytest.mark.repo_dirs(*directories)
+def test_new_sub_dir_absolute_path(repo: OnyoRepo) -> None:
+    """
+    Test that `onyo new --path <path>` can create an asset in sub-directories of
+    the repository while the cwd is inside a different sub-directory.
+    """
+    # Change cwd to sub-directory
+    cwd = repo.git.root / 'spe\"cial\\char\'acteஞrs'
+    os.chdir(cwd)
+
+    # build a path for the new asset as an absolute path
+    file = repo.git.root / "just another path" / "laptop_apple_macbookpro.0"
```

Comment:
I'd rename `file` for `file_` or `path`. Generally not a fan of shadowing builtins.

Reply:
Thanks a lot, I changed that for all tests of `test_new.py`!